script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -503,35 +499,34 @@ Does this immortality stuff have anything to do with Arcueid's enemy?"
 [sha:79b9c3ac1e9d1eea3ac15b17408f2c1c0ba6ca99]{
 -- Page 705, Offset 26166.
 -- 「ああ。ちょいと遠回りだがな。もうちっと我慢しろ。
-"Yeah. I'm still getting to that. Hold your horses.
+"Right, still getting to that. Hold your horses.
 }
 [sha:64604d68684ef8031cbb9ef352bbd04bc0979f88]{
 -- Page 705, Offset 26167.
 -- とにかく死徒と区別される吸血鬼の『不老不死』は不安定なものなんだよ。ヤツらの場合、単純に寿命が人間の数倍になった、という事だけだからな」
-As I was saying, the 'immortality' of Dead Apostle ranked vampires is a fragile thing.
- At the end of the day, they're just creatures with a lifespan several times longer than that of a human."
+Watcha need to know is that the 'immortality'' of these vampires we classify as Dead Apostles is real uncertain.
+ These guys... their lifespan have just gotten a buncha times longer than a human, that's all."
 }
 [sha:362c0bda01e766d32ed6c5347dd2be4a3eabf29a]{
 -- Page 705, Offset 26168.
 -- 「……単純にって。あいつら、寿命だけじゃなくて力とかも人間離れしてるだろ」
-// VERY liberally TL'd
-"...What do you mean 'just' their lifespan? What about their superhuman strength?"
+"What do you mean 'that's all'? It's not just their lifespan that's completely superhuman, but their abilities too.
 }
 [sha:2607d0018f10c65d367533437189891cf8670901]{
 -- Page 706, Offset 26169.
 -- 「おう、そこだ。
-"Yeah, about that.
+"Yeah, 's what I mean.
 }
 [sha:9fe9ee656f2aa8fc2b7c8cfd8e9b3195e6affc07]{
 -- Page 706, Offset 26170.
 -- 死徒の固有能力はヤツらが人間だった頃に手に入れたモノを、そのまま何百年と成長させた<異能|スキル>にすぎない。
-The unique abilities of a Dead Apostle are just the skills they had as a human, refined over hundreds of years.
+The abilities of Dead Apostles ain't nothing more than some skill they had when human that's been refined over a few hundred of years.
 }
 [sha:840e786546fb0fc1fceef3a49e31ae7f0b0c151d]{
 -- Page 706, Offset 26171.
 -- ヤツらは自身が学んだものを吸血鬼になっても学び続け、結果としてそれが原理になっただけだ」
-They continue their pursuit of knowledge even after they've become vampires,
- and in the end, it becomes their Principle."
+So they just took somethin' they'd been studying and then, after turning into a Dead Apostle,
```

Comment:
So they just took somethin' they'd been studyin', and then after turning into a Dead Apostle, kept at it until it became a Principle.